mllib/src/main/scala/org/apache/spark/mllib/feature/Word2Vec.scala

```diff
@@ -431,6 +431,14 @@ class Word2Vec extends Serializable with Logging {
 class Word2VecModel private[mllib] (
     private val model: Map[String, Array[Float]]) extends Serializable with Saveable {
```

Comment:
It seems to me that `Map[String, Int]` will just be `model.zip.(0 until model.size).toMap`. Is it right to expect that the ordering of keys in the model Map does not change?